app/server/src/models/userProfile/userProfile.model.js

```diff
@@ -0,0 +1,100 @@
+const mongoose = require("mongoose");
```

Comment:
We can delete this file, I integrated its fields to user model

Reply:
Deleted